syntax = "proto2";
package rrpc.test;

option cc_generic_services = true;

message EchoRequest {
    optional string message = 1;
}

message EchoResponse {
    optional string message = 1;
}

service EchoService {
    rpc Echo(EchoRequest) returns (EchoResponse);
}
